### PR TITLE
Prep for release 0.7.0

### PR DESCRIPTION
**Pull request by nturinski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+### Added
+- Project validation prior to `Create Static Web App...`
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
+- GitHub Configurations tree item when SWA is connected to local project
+- `Create Configuration File` command to add `staticwebapp.config.json`
+- Standard SKU for `Create Static Web App... (Advanced)`
+
+### Changed
+- `Create Static Web App...` requires an opened workspace in VS Code
+- Location, API location, GitHub org, and repo privacy are defaulted in `Create Static Web App...`
```
Comment (wwlorey) [inline CHANGELOG.md:13]: "defaulted" sounded a bit weird to me, maybe this?

```suggestion
- Location, API location, GitHub org, and repo privacy are automatically set in `Create Static Web App...`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
```
Comment (wwlorey) [inline CHANGELOG.md:5]: I would add something about git. This makes me think we validate their app before creating
```suggestion
- Git project validation prior to `Create Static Web App...`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
```
Comment (wwlorey) [inline CHANGELOG.md:6]: > workflow config

I don't know why I waited until now to look up the "correct" term for the workflow file... but it looks to be "GitHub Actions workflow": https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+- GitHub Configurations tree item when SWA is connected to local project
+- `Create Configuration File` command to add `staticwebapp.config.json`
+- Standard SKU for `Create Static Web App... (Advanced)`
```
Comment (wwlorey) [inline CHANGELOG.md:9]: nit: I wanna make sure it doesn't sound like only the standard sku is supported in advanced create
```suggestion
- Standard SKU option for `Create Static Web App... (Advanced)`
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
+- GitHub Configurations tree item when SWA is connected to local project
```
Comment (wwlorey) [inline CHANGELOG.md:7]: nit: the tree item name is just "configuration"
```suggestion
- GitHub Configuration tree item when SWA is connected to local project
```

File: CHANGELOG.md (hunk 1 above)
Comment (nturinski) [reply CHANGELOG.md:13]: More wordy, but I feel like using the word default feels kind of important to me 🤔💭
```suggestion
- Location, API location, GitHub org, and repo privacy steps have default values for `Create Static Web App...`
```

Comment (wwlorey) [reply CHANGELOG.md:13]: I dig it 👌

File: CHANGELOG.md (hunk 3 above)
Comment (nturinski) [reply CHANGELOG.md:6]:
```suggestion
- After creating a Static Web App, `git pull` is executed to retrieve the [GitHub Actions workflow file](https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow)
```

**Review by wwlorey — APPROVED** [review]

**Pull request merged**